internal/fabric_proxy.go

```diff
@@ -27,11 +28,21 @@ func NewFabricProxy(contextStore *ContextStore) *FabricProxy {
 }
 
 // FabricCall is the waPC HostCall function for interacting with the ledger
-func (proxy *FabricProxy) FabricCall(ctx context.Context, binding, namespace, operation string, payload []byte) ([]byte, error) {
+func (proxy *FabricProxy) FabricCall(ctx context.Context, binding, namespace, operation string, payload []byte) (result []byte, err error) {
```

Comment:
why have you added parameter names to the return signature?

Reply:
I guess because the `err` return value is being set by the deferred function

Reply:
Yes, it's for the deferred function to set err